### PR TITLE
Structuring for Wamp/Xampp users

### PR DESCRIPTION
**Pull request by acekyd**
Customized links to support the mentioned stacks, set xdebug limit to work with these stacks

### CODE REVIEW

File: app/Http/Controllers/PageController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+    namespace App\Http\Controllers;
+    use App\Http\Controllers\Controller;
+    use Illuminate\Http\Request;
```
Comment (unicodeveloper) [inline app/Http/Controllers/PageController.php]: This is not needed. Please remove.

File: app/Http/Controllers/PageController.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+    namespace App\Http\Controllers;
+    use App\Http\Controllers\Controller;
```
Comment (unicodeveloper) [inline app/Http/Controllers/PageController.php]: This is also not needed. Please remove. The controller it inherits is already in the same namespace

File: app/Http/Controllers/PageController.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        {
+            return view('apidashboard');
+
+        }    
+
+        public function contact()
+        {
+            return view('contact');
+
+        }
```
Comment (unicodeveloper) [inline app/Http/Controllers/PageController.php]: Please add comments. Same way comments are in the other controller files

**Comment by unicodeveloper** [comment]
Thanks @acekyd . Please fix the little issues and we'll be good to go. 😊

**Comment by acekyd** [comment]
Done @unicodeveloper

**Comment by unicodeveloper** [comment]
Awesome!, Merging it now!

**Pull request merged**